dts/bindings/input/gpio-keys.yaml

```diff
@@ -41,6 +41,12 @@ properties:
       Do not use interrupts for the key GPIOs, poll the pin periodically at the
       specified debounce-interval-ms instead.
 
+  no-disconnect:
+    type: boolean
+    description: |
+      Do not try to disconnect the pin on suspend, use of the gpio controller
+      does not support confguring pins in GPIO_DISCONNECTED mode.
```

Comment:
Typo `configuring` and I think you want to have two sentences?

Reply:
Oh wow, sorry wrote that on an airplane, I blame a turbulence, reworded. :-)